### ISSUE TITLE
Error:transformClassesWithRetrolambdaForRelease

### ISSUE THREAD

**Issue by ouyangzn** [evant/gradle-retrolambda#214]
Hi,
i got this error:
```
Error:Execution failed for task ':app:xxx:transformClassesWithRetrolambdaForRelease'.
> Could not extract variant from output dir: E:\xxx\xxx\xxx\xxx\app\xxx\build\intermediates\transforms\retrolambda\release\folders\1\1\retrolambda
```
I used:
```
classpath 'me.tatarka:gradle-retrolambda:+'
```
gradle version:
```
dependencies {
    classpath 'com.android.tools.build:gradle:2.2.0'
    // retrolambda (https://github.com/evant/gradle-retrolambda)
    classpath 'me.tatarka:gradle-retrolambda:+'
  }

task wrapper(type: Wrapper) {
  gradleVersion = '2.9'
  distributionUrl = 'https://services.gradle.org/distributions/gradle-2.8-all.zip'
}
```

**Comment by evant** [comment]
In case you didn't know, you are pointing to `3.3.1-SNAPSHOT`. It apperas it wasn't working correctly on windows. I have uploaded another build that should fix the issue. Can you re-test?

**Comment by ouyangzn** [comment]
It's still not work， the same error. -_-!

**Comment by evant** [comment]
Thanks, I actually spun up a windows environment to test this. Should be fixed in `3.3.1`.
